docs/site/submitting_a_pr.md

```diff
@@ -55,22 +58,14 @@ npm run lint:fix && npm test
 
 {% include note.html content="<b>npm run lint:fix</b> might reformat the source code and fix style issues. Be sure to add such changes to your commit." %}
 
-### 3. After creating PR, sign the CLA and fill out checklist
-
-After creating the pull request, make sure the Contributor License Agreement
-(CLA) has been signed for the `loopback-next` repository.
+### 3. After creating PR, fill out checklist
 
 The pull request has a checklist that must be filled out appropriately.
 
 ![submit_pr_template_1.png](./imgs/submit_pr_template_1.png)
```

Comment:
Please update the screenshot too, here is what I see in your PR today:


![](https://github.com/strongloop/loopback-next/raw/9911a348ae9957e036084c02364e36437774e345/docs/site/imgs/submit_pr_template_1.png)

Reply:
good catch!